internal/db/migrations/005_reservations.sql

```diff
@@ -23,9 +23,11 @@ CREATE TABLE aws_reservation_details
   provider INTEGER NOT NULL DEFAULT provider_type_aws(),
   pubkey_id BIGINT NOT NULL REFERENCES pubkeys(id),
   source_id TEXT NOT NULL,
+  name TEXT NOT NULL DEFAULT '',
```

Comment:
I've got a bad news, that this won't be enough, we will want to have name per instance, as you've suggested and research shared with me yesterday confirmed.

Reply:
Sure, let's utilize an array, postgres can easily do that and since it is not indexed column and will never be, no problem.

I am wondering if we should think about serializing all the reservation details into JSON, I find those columns harder and harder to maintain as it grows. We really only need the info to be able to return the information back to the user when they request reservation info (something to be displayed on the future pending reservations page). What you think @ezr-ondrej @adiabramovitch ?

Serializing into JSON is easy, postgres even have a native type for that and we could easily add more info. On the other hand, it brings one big negative - compatibility. A reservation after application upgrade might be out of date so unmarhsalling of old record could fail. On the other hand, this type of table is a good candidate for that because we will regularly delete old reservations so these kinds of bugs will naturally disappear as old reservations will get deleted after few days.